spec/unit/shoulda/matchers/active_model/numericality_matchers/comparison_matcher_spec.rb

```diff
@@ -40,6 +44,11 @@ class << fake_matcher
 
   context 'greater_than_or_equal_to' do
     it do
+      expect(instance_with_validations(greater_than_or_equal_to: 2, strict: true))
+        .to matcher.is_greater_than_or_equal_to(2).strict
```

Comment:
This is true and same for the other warnings just like this. Not sure how the bad style crept through for other tests.